doc/storage.md

```diff
@@ -0,0 +1,54 @@
+## Using the Storage module
+
+**This document does not apply to apps that are migrated from v1 of the Zendesk
+App Framework.**
+
+The Storage module provides helper methods to interact with `localStorage`. It is important to note that different installations of the same app have the same url, therefore they will share `localStorage` on the same browser even across different accounts. The storage module scopes `localStorage` keys using the key passed to its constructor. You can pass the installation id to the storage constructor to prevent conflicts between different installations.
```

Comment:
> different installations of the same app have the same url

only applies to apps that are hosted on zdusercontent.com, not necessarily self-hosted apps